content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +46,392 @@ permissions:
 }
 ```
 
-> Note: The `route53:ListHostedZonesByName` statement can be removed if you
+> ℹ️ The `route53:ListHostedZonesByName` statement can be removed if you
 > specify the (optional) `hostedZoneID`. You can further tighten the policy by
 > limiting the hosted zone that cert-manager has access to (e.g.
 > `arn:aws:route53:::hostedzone/DIKER8JEXAMPLE`).
+>
+> 📖 Read about [actions supported by Amazon Route 53](https://docs.aws.amazon.com/Route53/latest/APIReference/API_Operations_Amazon_Route_53.html),
+> in the [Amazon Route 53 API Reference](https://docs.aws.amazon.com/Route53/latest/APIReference/Welcome.html).
+>
+> 📖 Learn how [`eksctl` can automatically create the cert-manager IAM policy](https://eksctl.io/usage/iam-policies/#cert-manager-policy), if you use EKS.
 
 ## Credentials
 
-You have two options for the set up - either create a user or a role and attach
-that policy from above.  Using a role is considered best practice because you do
-not have to store permanent credentials in a secret.
+cert-manager needs an [AWS access key](https://docs.aws.amazon.com/glossary/latest/reference/glos-chap.html#access_key),
+to authenticate to the Route53 API.
+An access key is defined by AWS as follows:
+> **access key**:
+> The combination of an access key ID (for example, `AKIAIOSFODNN7EXAMPLE`) and a secret access key (for example, `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY`). You use access keys to sign API requests that you make to AWS.
+
+You have two options:
+1. (Best Practice) Use an [IAM Role with temporary security credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#bp-workloads-use-roles).
+2. Use an [IAM User with a long-term access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+
+Using an IAM Role with temporary security credentials is considered best practice because:
+1. You do not have to store the long-term access key (e.g. in a Secret)
+2. You don't have to manage [access key rotation](https://docs.aws.amazon.com/glossary/latest/reference/glos-chap.html#keyrotate).
+
+Using an [IAM User with long-term access key](#iam-user-with-long-term-access-key) is a reasonable choice if cert-manager
+is deployed outside AWS but the DNS zone is on Route53.
+
+cert-manager supports multiple ways to get the access key and these can be categorized as either "ambient" or "non-ambient".
+
+### Ambient Credentials
```

Comment:
In https://github.com/cert-manager/cert-manager/issues/4738 @twe-syde requested better "Documentation about ambient-credentials" and that: 
> The documentation should mentioned at least these two flags and why they are needed for IRSA. 
> --issuer-ambient-credentials=true|false
> --clusterissuer-ambient-credentials=true|false

I've tried to address that here. 
 * https://deploy-preview-1555--cert-manager.netlify.app/docs/configuration/acme/dns01/route53/#ambient-credentials

@twe-syde Please give feedback if you have time.